addons/mail/static/src/xml/followers.xml

```diff
@@ -9,7 +9,7 @@
             <div class="o_followers_title_box">
                 <button class="btn btn-link dropdown-toggle" title="Dropdown menu - Followers"
                         data-toggle="dropdown" aria-expanded="false">
-                    <i class="fa fa-user" role="img" aria-label="Dropdown menu - Followers"></i>
+                    <i class="fa fa-user" role="img" aria-label="Followers"></i>
```

Comment:
I've just seen that aria-label is not translated in JS views so it's not necessary to export the translation, but that will be the case when this PR is merged:

https://github.com/odoo/odoo/pull/48255

Reply:
Done, I exported that term. BTW, thanks for that PR!